docs/UserGuide.md

```diff
@@ -235,6 +408,15 @@ Example of usage:
 1. `add -m CS2113T`
 2. `undo`
 
+Example of output:
+```
+Command: 
+⋗	undo
+════════════════════════════════════════════════════════════════════════════════
+Undo successful.
+════════════════════════════════════════════════════════════════════════════════
+```
+
```

Comment:
Maybe show the output of instead
1. `add -m CS2113T`
2. `list -m`
3. `undo`
4. `list -m`
Or it would be wise to show the exact command the undo reverted